oauth2client/contrib/gce.py

```diff
@@ -48,28 +45,67 @@
 """
 
 
-def _get_service_account_email(http_request=None):
-    """Get the GCE service account email from the current environment.
-
+def _get_metadata(http_request=None, path=None, recursive=True):
```

Comment:
Just because it's inconsistent and it requires the trailing `/` and gave me some frustrating bugs. No actual qualms.
`access_token` and `expires_in` are the only fields in the response from the `.../token` endpoint, and we use both of them.